src/test/java/jp/ikedam/jenkins/plugins/updatesitesmanager/ManagedUpdateSiteJenkinsTest.java

```diff
@@ -96,7 +95,7 @@ public void shouldReturnValidationErrOnWrongCert(String cert) {
     @WebServerRecipe
     public void shouldFailUpdateWithoutCert() throws Exception {
         // Ensure to use server-based download.
-        DownloadSettings.get().setUseBrowser(false);
+//        DownloadSettings.get().setUseBrowser(false);

```

Comment:
Change request: You can just remove unused codes.

Reply:
Ok I was not sure because that class does not exist anymore starting from Jenkins 2.200